trio_websocket/__init__.py

```diff
@@ -22,21 +22,46 @@
 
 @asynccontextmanager
 @async_generator
-async def open_websocket(host, port, resource, use_ssl, nursery=None):
+async def open_websocket(host, port, resource, use_ssl):
```

Comment:
Default false on use_ssl to throughout api?

Reply:
I'm a security nerd, so I made the unusual/controversial decision to require the SSL argument. This forces the user to at least think about whether to use SSL or not.